### ISSUE TITLE
[Layout] Provide a filter to easily find an app from the app registry

### ISSUE THREAD

**Issue by BalouBalou** [Meeds-io/meeds#2353]
**Steps**
- when editing a page
- say you search for an app to include in the page
- you need to browse different categories without knowing where your app is

**Suggested**
- Add a filter in the app registry so it is easy for users to find what they're looking to
- Filter in all or in a category if a category is displayed
- filter by name or by description

**Comment by srenault-meeds** [comment]
Hi @BalouBalou You already can access the name of the app using the layout editor.
Indeed, when editing the application, the drawer's header is displaying the name's application.
See screenshot:
<img width="518" alt="image" src="https://github.com/user-attachments/assets/8646c3ab-367d-4041-9dba-f9dc0a9ee79d">

Is this what you are looking for?

**Comment by BalouBalou** [comment]
Thanks for your feedback @srenault-meeds .
FYI We've been several colleagues who didn't have the right reflex to get the information. 
But now we know.

Nevertheless, It could be great to have a filter function in a long term, in order to save clicks and reduce search time.
When I want to add an image, I only have to start writing "Image" and I choose my app.
But this would be a personal nice to have function.

It's already great to see the name of the app.

Thanks for your help


Have a great day

**Comment by srenault-meeds** [comment]
yes indeed, a text filter within the applications drawer would be nice
